lib/loader/loader.c

```diff
@@ -121,3 +122,26 @@ evmc_create_fn evmc_load(const char* filename, enum evmc_loader_error_code* erro
         *error_code = ec;
     return create_fn;
 }
+
+struct evmc_instance* evmc_load_and_create(const char* filename, enum evmc_loader_error_code* error_code)
+{
+    evmc_create_fn create_fn = evmc_load(filename, error_code);
```

Comment:
I think it should be safe (at least on POSIX) to call this multiple times and not pollute the memory (since `dlopen` should just return the reference if it was loaded already IIRC). However, is that the case for Windows?

Do we need to maintain our local list of loaded EMVC libraries in order to make this helper callable multiple times?

Reply:
It's the same of Windows.

>  Calling LoadLibrary increments the reference count.

Reply:
It should be documented then that calling `evmc_load` and `evmc_load_and_create` multiple times should be safe and not blow up memory usage.